proc_netstat.go

```diff
@@ -32,7 +32,7 @@ type ProcNetstat struct {
 	IpExt
 }
 
-type TcpExt struct {
+type TcpExt struct { // nolint:revive
```

Comment:
Since this was just recently added, should we just change it to TCPExt so we don't need to skip linting?  And similar question for the other recently added lint failures.

Reply:
I was thinking about following up with a big breaking change to clean up the nolint.